controllers/nf/smf/reconciler.go

```diff
@@ -205,12 +190,12 @@ func (r *SMFDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			}
 		} else {
 			/*
```

Comment:
Can we clean the code up by removing commented code?

Reply:
We kept commented because it is commented in R1 but we can remove the comments.